.github/workflows/deploy-pages.yaml

```diff
@@ -0,0 +1,44 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Deploy static content to Pages
+
+on:
+  # Run on pushes targeting the default branch
+  push:
+    branches:
+      - master
+
+  # Allow this workflow to be run manually from the Actions tab
+  workflow_dispatch:
+
```

Comment:
Double-checking that this is intended to be empty?

Reply:
I assume so; it was empty in the GitHub-provided template.

I reworded some comments and changed the branch list from `["master"]` to `\n  - master`. Otherwise this is identical to the provided template.

Reply:
Seems reasonable to me